packages/sit-onyx/src/components/OnyxFileUpload/examples/FileUploadActionsExample.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+import { iconMoreVertical, iconTrash } from "@sit-onyx/icons";
+import { ref } from "vue";
+import OnyxFileCard from "../../OnyxFileCard/OnyxFileCard.vue";
+import OnyxIconButton from "../../OnyxIconButton/OnyxIconButton.vue";
+import OnyxFlyoutMenu from "../../OnyxNavBar/modules/OnyxFlyoutMenu/OnyxFlyoutMenu.vue";
+import OnyxMenuItem from "../../OnyxNavBar/modules/OnyxMenuItem/OnyxMenuItem.vue";
+import OnyxSystemButton from "../../OnyxSystemButton/OnyxSystemButton.vue";
+import OnyxFileUpload from "../OnyxFileUpload.vue";
+
+const allFiles = ref<File[]>([]);
+
+const removeFile = (fileToRemove: File) => {
+  allFiles.value = allFiles.value.filter((file) => file !== fileToRemove);
+};
+</script>
+
+<template>
+  <OnyxFileUpload v-model="allFiles" :multiple="true" list-type="hidden">
```

Comment:
```suggestion
  <OnyxFileUpload v-model="allFiles" :multiple="true">
```

With type hidden nothing is displayed. If we want to show custom file cards, we need to remove the list type